taxidermy.php

```diff
@@ -29,290 +29,302 @@
 <html lang="en-GB">
     <head>
         <title>Taxidermy Collection</title>
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
     </head>
-    <header>
-        <h1>Life in Death</h1>
-        <h2>Curios & Oddities</h2>
-    </header>
-    <section id="intro">
-        <p>Here is a nice gallery of my Taxidermy Collection so far.
-        Items are sorted by their Name/ID and include details about their origin,
-        as well as an overall rating out of 10.</p>
-    </section>
-    <section class="collection">
-        <div class="collection_item">
-            <img src="<?//image path
-            $imageVariable = $imageQuery->fetch();
-            foreach($imageVariable as $value) {
-                echo $value;
-            } ?>"  alt="<? //description
-            $descriptionVariable = $descriptionQuery->fetch();
-            foreach($descriptionVariable as $value) {
-                echo $value;
-            } ?>">
-            <h3><? //just the name value
-                $nameVariable = $nameQuery->fetch();
-                foreach($nameVariable as $value) {
+    <body>
+        <header>
+            <h1>Life in Death</h1>
+            <h2>Curios & Oddities</h2>
+        </header>
+        <section id="intro">
+            <p>Here is a nice gallery of my Taxidermy Collection so far.
+                Items are sorted by their Name/ID and include details about their origin,
+                as well as an overall rating out of 10.</p>
+        </section>
+        <section class="collection">
+            <div class="collection_item">
+                <img src="<?//image path
+                $imageVariable = $imageQuery->fetch();
+                foreach($imageVariable as $value) {
                     echo $value;
-                } ?></h3>
```

Comment:
It might be the way github is configured, but I'm not sure what's supposed to be in this foreach loop?